Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/lucene/impl/LuceneUtility.java

```diff
@@ -132,19 +146,24 @@
     private Directory index;
     private Path relativePath = null;
     private IndexSearcher searcher = null;
-
+    private Analyzer customAnalyzer = null;
+    private Query query = null;
```

Comment:
what's that used for?

Reply:
remove the query variable which isn't being used anywhere. The customAnalyzer is used by add ascii folding filter to it. The variable is declared there